Makefile

```diff
@@ -9,10 +9,11 @@ GOLIST_PKG=$(GOLIST) ./... | grep -v /vendor/
 GOLINT=golint
 GORELEASER=goreleaser
 BINARY_NAME=sd-local
+COVERPROFILE?=cover.out
```

Comment:
```suggestion
COVERPROFILE=cover.out
```

Reply:
I don't think it's a problem because it's a variable declaration method that uses default values in Makefiles.

Reply:
I didn't know that, thanks.